### DISCUSSION TITLE
Dynamic Routes, DEV mode, `pages/test/[test].js` timesout

### DISCUSSION THREAD

**Discussion by wibed** [vercel/next.js#28021] [Help]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

16.5.0

### What browser are you using?

Firefox

### What operating system are you using?

Linux

### How are you deploying your application?

npm run dev

### Describe the Bug

navigating to a dynamic route in dev mode results in a timeout

### Expected Behavior

dynamic routes to work in dev mode

### To Reproduce

1. create default nextjs app
2. create additional page with slug as described here https://nextjs.org/docs/routing/dynamic-routes
3. start the app in dev mode
4. navigate to the page, expirience timeout

**Comment by timneutkens** [comment]
Please add a complete reproduction to your report, this can't be reproduced based on the steps provided.

**Reply by wibed** [reply]
it started working again, after recreating the project and copying my code into it. 
simply upgrading the project and deleting temporary files and folders, did not help.
